<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="myTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_01">
     <attribute name="title">
      <string>Tab 1, Test</string>
     </attribute>
     <widget class="QPushButton" name="homePushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetNameButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Device Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GetPoseButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Pose</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReadDevNameButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Device Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SpecPoseButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Spec</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ToHOMeButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>ToHOME</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_02">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="GetPoseButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>GetPose</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FloatXYZRlineEdit">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="FloatXYZRlabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Float values X, Y, Z, R</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DecimalXYZRlineEdit">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="DecimalXYZRlabel">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Decimal values X, Y, Z, R</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FloatXYZRlineEdit_Y">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>170</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FloatXYZRlineEdit_Z">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>280</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="FloatXYZRlineEdit_R">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>350</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GetHOMeParamsButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>GetHOMEParams</string>
      </property>
     </widget>
     <widget class="QPushButton" name="XAdd10Button">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>160</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>X + 10</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
